addons/l10n_es/data/template/account.tax-es_canary_common.csv

```diff
@@ -243,3 +243,27 @@
+"","","","","","","","","","","","","tax","invoice","account_common_473","","-mod130[06]","",""
+"","","","","","","","","","","","","base","refund","","","","",""
+"","","","","","","","","","","","","tax","refund","account_common_473","","+mod130[06]","",""
+"account_tax_template_p_irpf15","1000","True","purchase","15% Withholding income tax","Retenciones IRPF 15%","15% WHI","","-15.0","percent","tax_group_retenciones_15","","base","invoice","","","+mod111[08]","fp_irpf15_canary","account_tax_template_igic_sop_0_inv,account_tax_template_igic_sop_0,account_tax_template_igic_sop_3_inv,account_tax_template_igic_sop_3,account_tax_template_igic_sop_7_inv,account_tax_template_igic_sop_7,account_tax_template_igic_sop_9_5_inv,account_tax_template_igic_sop_9_5,account_tax_template_igic_sop_15_inv,account_tax_template_igic_sop_15,account_tax_template_igic_sop_20_inv,account_tax_template_igic_sop_20"
+"","","","","","","","","","","","","tax","invoice","account_common_4751","","-mod111[09]","",""
+"","","","","","","","","","","","","base","refund","","","-mod111[08]","",""
+"","","","","","","","","","","","","tax","refund","account_common_4751","","+mod111[09]","",""
+"account_tax_template_s_irpf19a","1000","True","sale","19% Withholding (leases)","Retenciones a cuenta 19% (Arrendamientos)","19% WH lease","Retención 19% (Arrend.)","-19.0","percent","tax_group_retenciones_19","","base","invoice","","","","fp_irpf19a_canary","account_tax_template_igic_r_0,account_tax_template_igic_r_3,account_tax_template_igic_r_7,account_tax_template_igic_r_9_5,account_tax_template_igic_r_15,account_tax_template_igic_r_20"
+"","","","","","","","","","","","","tax","invoice","account_common_473","","-mod130[06]","",""
+"","","","","","","","","","","","","base","refund","","","","",""
+"","","","","","","","","","","","","tax","refund","account_common_473","","+mod130[06]","",""
+"account_tax_template_p_irpf19a","1000","True","purchase","19% Withholding (leases)","Retenciones 19% (Arrendamientos)","19% WH lease","","-19.0","percent","tax_group_retenciones_19","","base","invoice","","","+mod115[02]","fp_irpf19a_canary","account_tax_template_igic_sop_0_inv,account_tax_template_igic_sop_0,account_tax_template_igic_sop_3_inv,account_tax_template_igic_sop_3,account_tax_template_igic_sop_7_inv,account_tax_template_igic_sop_7,account_tax_template_igic_sop_9_5_inv,account_tax_template_igic_sop_9_5,account_tax_template_igic_sop_15_inv,account_tax_template_igic_sop_15,account_tax_template_igic_sop_20_inv,account_tax_template_igic_sop_20"
+"","","","","","","","","","","","","tax","invoice","account_common_4751","","-mod115[03]","",""
+"","","","","","","","","","","","","base","refund","","","-mod115[02]","",""
+"","","","","","","","","","","","","tax","refund","account_common_4751","","+mod115[03]","",""
```

Comment:
@h4818 Only 6 taxes?

Reply:
These are the only ones that had a mapping...
but CHKL said we should add them all back. I will change it.